bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -611,26 +612,10 @@ public boolean isAborted() {
 
   @Override
   public void truncateTable(TableName tableName, boolean preserveSplits) throws IOException {
```

Comment:
Please add a comment that preserveSplits is ignored, and splits will remain in place until Bigtable cleans them up.